HybrasylXML/XSD/hybrasylExtensions.cs

```diff
@@ -312,6 +312,32 @@ namespace Hybrasyl.Creatures
     public partial class Spawn
     {
         protected static Random Rng = new Random();
```

Comment:
not your code, but we should probably change `Rng` here to be an instance variable